fusor-ember-cli/app/templates/review/installation.hbs

```diff
@@ -194,11 +194,15 @@
                                   value=sub.contract_number }}
                     {{review-link label=qtyLabel
                                   routeName='subscriptions.select-subscriptions'
-                                  value=sub.quantity_to_add }}
+                                  value=(if isMissingSubscriptions null sub.quantity_to_add)
+                                  isRequired=true
+                                  validationMessage="Need to re-enter"}}
```

Comment:
If the "Deploy" button is now disabled when `isMissingSubscriptions` is `true`, then I need to show in the review page what is missing in red, so it will say "Need to re-enter" and the user is taken to the `select-subscriptions` route.